tests/integration/templates/cdktemplate.json

```diff
@@ -0,0 +1,579 @@
+{
```

Comment:
Can we also check in the CDK source file(s) that were used to generate this template? (This will make it easier to extend the template with additional resources in the future..)